### PR TITLE
Snippets related change

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/264/

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

https://github.com/PureStake/moonbeam-docs/pull/264/

### CODE REVIEW

File: builders/interact/eth-libraries/deploy-contract.md [hunk 1]
```diff
@@ -192,7 +184,7 @@ npm init --yes
 
 ### Web3.py {: #web3py } 
 
-In the first part of [the script](/snippets/code/web3py-contract/deploy.py), the `web3` instance (or provider) is created using the 在[此脚本](/snippets/code/web3py-contract/deploy.py)的第一部分，`web3`实例（或是提供者）是使用提供者RPC的`Web3(Web3.HTTPProvider(provider_rpc))`方式所创建的。您可以通过改变提供者RPC，选择要将交易发送到您指定的网络。
+In the first part of the script, the `web3` instance (or provider) is created using the 在此脚本的第一部分，`web3`实例（或是提供者）是使用提供者RPC的`Web3(Web3.HTTPProvider(provider_rpc))`方式所创建的。您可以通过改变提供者RPC，选择要将交易发送到您指定的网络。
```
Comment (eshaben) [inline builders/interact/eth-libraries/deploy-contract.md:187]: some of this is in English.. and it also looks like there might be an extra `web3` in there

File: builders/interact/eth-libraries/deploy-contract.md [hunk 2]
```diff
@@ -290,11 +274,7 @@ In the first part of [the script](/snippets/code/web3py-contract/deploy.py), the
     --8<-- 'code/web3py-contract/increment.py'
     ```
 
-其次，用来合约交互的文件是 _reset.*_ ，该文件的功能是将合约中存储的数字重置归零。您可以通过以下链接找到对应库的代码段（文件被命名为`reset.*`）：
-
- - Web3.js: [_reset.js_](/snippets/code/web3-contract-local/reset.js)
- - Ethers.js: [_reset.js_](/snippets/code/ethers-contract-local/reset.js)
- - Web3.py: [_reset.py_](/snippets/code/web3py-contract/reset.py)
+其次，用来合约交互的文件是 _reset.*_ ，该文件的功能是将合约中存储的数字重置归零。您可以创建一个名为 `increment.js` 或 `increment.py` 的文件，具体取决于您使用的语言。
```
Comment (eshaben) [inline builders/interact/eth-libraries/deploy-contract.md:277]: I think this should be `reset.js`and `reset.py`

**Review by eshaben — CHANGES_REQUESTED** [review]
just a couple of small things

**Comment by hyd628** [comment]
Thanks. I pushed the fixes.

**Pull request merged**